config/dictionaries/resource.json

```diff
@@ -548,6 +548,31 @@
       }
     ]
   },
+  {
+    "id": "download",
+    "name": "Download balenaOS",
+    "fields": [],
+    "examples": [
+      {
+        "id": "download-private-balenaos",
+        "summary": "Download unmanaged releases of balenaOS for private device types",
+        "description": "Unmanaged releases of balenaOS for private device types are not released publically. If you have access to a private device type, then images can be downloaded by providing an API key. Add an `--output` flag to provide a path for the image to be downloaded in the curl command below.",
```

Comment:
> Unmanaged releases of balenaOS for private device types are not released publically.

That's a bit confusing. Yes they aren't publically but you could provide a bearer header and then it should work. So I would rephrase this like "in order to access private or ESR images you need to provide your bearer token".

Reply:
> I'm a bit against documenting this tbh and offering only the UI & the CLI as the ways to download images

I know, I know but we do provide a way and it's not like it's going anywhere. IMHO I feel it's better we document it. 
I will mention the requirement for sure, and will also mention that it's a stream. 
I like your version better, yeah. Coming up.